src/openInBrowser.js

```diff
@@ -0,0 +1,14 @@
+const opn = require('opn');
+
+const openInBrowser = (url, browser) => {
+  try {
+    const options = typeof browser === 'string' ? {app: browser} : undefined;
+
+    opn(url, options).catch(() => {}); // Prevent `unhandledRejection` error.
```

Comment:
If you make `openInBrowser` an async function you can await `opn`, so that any errors will get caught in the `try/catch`.

Reply:
ahhh right, i keep forgetting source is transpiled, i'll make the fix